src/test/java/org/wildfly/bot/utils/testing/model/GitHubEventPayload.java

```diff
@@ -0,0 +1,253 @@
+package org.wildfly.bot.utils.testing.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.smallrye.mutiny.tuples.Tuple2;
+import org.kohsuke.github.GHEvent;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @implNote {@code REPOSITORY}, {@code INSTALLATION} & {@code SENDER} attributes are
+ *           present in most events received by GitHub SSE. Thus, we generate them for easier testing.
+ */
+public abstract class GitHubEventPayload {
+    private static final String ID = "id";
+    private static final String TYPE = "type";
+    private static final String REPO = "repo";
+    private static final String REPO_NAME = "name";
+    private static final String PAYLOAD = "payload";
+    private static final String REPOSITORY = "repository";
+    private static final String INSTALLATION = "installation";
+    private static final String SENDER = "sender";
+    private static final String REPOSITORY_VALUE = """
```

Comment:
This is problem with mocking. The `MockSettings` in `quarkus-github-app` for mocking don't allow flag for serializing/deserializing mocks. I.e. when in user's code we want to do that, the test will fail. This can be observed in [GitHubEventPreprocessor](https://github.com/wildfly/wildfly-github-bot/pull/200/files#diff-587228e54a6ec2d70782878251d382cc1c7ed8422daf90e2800e5ca9fa324f46R54)